src/sentry/static/sentry/app/components/search/sources/index.jsx

```diff
@@ -26,7 +26,7 @@ class SearchSources extends React.Component {
     let foundResults = isLoading
       ? []
       : flatten(allSources.map(({results}) => results || [])).sort(
-          (a, b) => a.score - b.score
+          (a, b) => b.score - a.score
```

Comment:
@billyvg is this edit correct? (higher scores should come first)

Reply:
@adhiraj It's not because we do want lower scores first (`0` means complete match: http://fusejs.io/). Can we add a comment here?